Elemental flood counter

permcody opened the pull request.

permcody committed:
Adding initial support in flood counter for elemental flooding
refs #4881

permcody committed:
Here I'm attempting to add the periodic node information to the element packing.
This doesn't work though because there is an overlap of element ids and node ids.

permcody committed:
Workaround to avoid id collisions

permcody committed:
Adjustments to GrainTracker for elemental floods

permcody committed:
New elemental test

permcody committed:
We need to add periodic nodes both directions for the elemtnal case

moosebuild commented:
Results of testing 29d1b4126e390cb6c92be721d8b74dab30c25faa using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14119

moosebuild commented:
Results of testing 29d1b4126e390cb6c92be721d8b74dab30c25faa using moose_PR_app_tests recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14122

moosebuild commented:
Results of testing 29d1b4126e390cb6c92be721d8b74dab30c25faa using moose_PR_test recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14120

moosebuild commented:
Results of testing 29d1b4126e390cb6c92be721d8b74dab30c25faa using moose_PR_test_dbg recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14121

moosebuild commented:
Results of testing 625f5e0d66f5272108393513ab66aa2cd3aa06f3 using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14155

moosebuild commented:
Results of testing 625f5e0d66f5272108393513ab66aa2cd3aa06f3 using moose_PR_app_tests recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14158

moosebuild commented:
Results of testing 625f5e0d66f5272108393513ab66aa2cd3aa06f3 using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14156

moosebuild commented:
Results of testing 625f5e0d66f5272108393513ab66aa2cd3aa06f3 using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14157

permcody committed:
Elemental solution value remapping logic

moosebuild commented:
Results of testing 64664cef70557d3ad8da21b84c59770d783513b8 using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14165

moosebuild commented:
Results of testing 64664cef70557d3ad8da21b84c59770d783513b8 using moose_PR_app_tests recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14168

moosebuild commented:
Results of testing 64664cef70557d3ad8da21b84c59770d783513b8 using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14166

moosebuild commented:
Results of testing 64664cef70557d3ad8da21b84c59770d783513b8 using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14167

permcody committed:
Update the NodalFloodAux AuxKernel to handle new elemental flood.

permcody committed:
Make adaptivity work with elemental flood

permcody committed:
Changing the periodic merge algorithm. Instead of adding periodic information and
communicating, I'm now adding periodic information during the merge step. This
also required the modification of MooseMesh::buildPeriodicNodeMap()

moosebuild commented:
Results of testing e53a4b0b5b295be635f0cbb131d998abd5fd349f using moose_PR_pre_check recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14382

moosebuild commented:
Results of testing dad092a55b8c5c6f20444a31f1358deff8923d86 using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14391

moosebuild commented:
Results of testing dad092a55b8c5c6f20444a31f1358deff8923d86 using moose_PR_app_tests recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14394

moosebuild commented:
Results of testing dad092a55b8c5c6f20444a31f1358deff8923d86 using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14392

moosebuild commented:
Results of testing dad092a55b8c5c6f20444a31f1358deff8923d86 using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14393

permcody committed:
Here's the important update. This undoes all the Linear Lagrange hackery to get nodal information to the quadrature points. Now we'll just use our flood data directly for elemental values (i.e. op active maps)

moosebuild commented:
Results of testing a729f07b5e0435ec6f6678621907447ba44c6962 using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14421

moosebuild commented:
Results of testing a729f07b5e0435ec6f6678621907447ba44c6962 using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14422

moosebuild commented:
Results of testing a729f07b5e0435ec6f6678621907447ba44c6962 using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14423

moosebuild commented:
Results of testing a729f07b5e0435ec6f6678621907447ba44c6962 using moose_PR_app_tests recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/14424

permcody commented:
closing in favor of the integration branch #4906
